internal/cli/config/config.go

```diff
@@ -67,13 +74,47 @@ func (opts *configOpts) Save() error {
 }
 
 func (opts *configOpts) Run() error {
-	helpLink := "https://docs.atlas.mongodb.com/configure-api-access/"
+	fmt.Printf(`You are setting up %s
```

Comment:
Should we make this more obvious that we are setting up a profile?
```suggestion
	fmt.Printf(`You are configuring a profile for %s.
```